File: tbot-application/src/main/java/org/andreidodu/tbot/dto/BotDTO.java
package org.andreidodu.tbot.dto;

import java.util.ArrayList;
import java.util.List;

import lombok.Data;

@Data
public class BotDTO {

	private String codiceBot;
	private List<DomandaDTO> caratteristiche = new ArrayList<>();
	private List<UrlDTO> urls = new ArrayList<>();
	private List<String> codiciUrl = new ArrayList<>();
	
}

File: tbot-web/src/main/tbot-web/src/app/modules/control-panel/control-panel-home/dto/bot-dto.ts
import {UrlDTO} from './url-dto';
import {DomandaDto} from '../../../common/components/form-input/dto/domanda-dto';

export interface BotDTO {

  codiceBot: string;
  urls: Array<UrlDTO>;
  codiciUrls: Array<string>;
  _toggle: boolean;
  caratteristiche: Array<DomandaDto>;
  _nome?: string;
  _descrizione?: string;

}
